### ISSUE TITLE
The online web is not working

### ISSUE THREAD

**Issue by monya001** [jhnklly/projectionuesser#1]
Hello,

I try to use http://jhnklly.github.io/projectionuesser/ but it looks like there is a problem with it.
The Java Script Console shows next error:

dropzone.js:1482
POST http://jhnklly.github.io/projectionuesser/pyproj_transform_upload.py
net::ERR_CONNECTION_RESET

Please help,

**Comment by almccon** [comment]
Yes, it seems like this doesn't include instructions for setting up the PostGIS backend?
